src/modules/72-templates-library/components/Templates/StageTemplate/StageTemplate.tsx

```diff
@@ -25,6 +25,6 @@ export class StageTemplate extends Template<NGTemplateInfoConfig> {
   }
 
   renderTemplateCanvas(props: TemplateProps<NGTemplateInfoConfig>): JSX.Element {
-    return <StageTemplateCanvasWrapperWithRef ref={props.formikRef} />
+    return <StageTemplateCanvasWrapperWithRef ref={props.formikRef as any} />
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
    return <StageTemplateCanvasWrapperWithRef ref={props.formikRef as unknown} />
```